lib/create_poll.dart

```diff
@@ -5,10 +5,20 @@ class CreatePoll extends StatefulWidget {
   _CreatePollState createState() => _CreatePollState();
 }
 
+int _count = 1;
+List<String> optionsList = new List();
```

Comment:
Where do you use these two variables?

Reply:
let it be, can be used in next iteration.